app.ipynb

```diff
@@ -57,14 +57,14 @@
     "    # Percentage => Base value + percentage\n",
     "    calculation_type = pn.widgets.RadioButtonGroup(options=['Absolute', 'Relatif', 'Percentage'], align='end')\n",
     "    # Replacement value\n",
-    "    spinner = pn.widgets.Spinner(name='Replacement Value', value=0, align='start')\n",
+    "    spinner = pn.widgets.IntInput(name='Replacement Value', value=0, align='start')\n",
```

Comment:
We need to pin a minimum version of panel to 0.10 to ensure this works correctly for further use